torch/csrc/jit/ir.cpp

```diff
@@ -178,8 +178,12 @@ void Node::printAttrValue(std::ostream& out, const Symbol& name) const {
   }
 }
 
-void Node::printAttributes(std::ostream& out, bool ignore_subgraph = false)
-    const {
+void Node::printAttributes(std::ostream &out, bool ignore_subgraph = false,
+                           bool skip) const {
+
+  if (skip) {
```

Comment:
This looks strange, can we just check if we want to call `printAttributes` at the callsites?